### ISSUE TITLE
[0.8.2 staging-13-d] Fishing pole line break

### ISSUE THREAD

**Issue by beastyn** [StrangeLoopGames/EcoIssues#12389]
In 7.8.9 Fishing line is solid

![bandicam 2019-05-23 14-24-31-390](https://user-images.githubusercontent.com/27898520/58241490-869bbb80-7d66-11e9-908d-f2f9524f53e7.jpg)

But from 8.1 we have this

![bandicam 2019-05-23 14-26-12-342](https://user-images.githubusercontent.com/27898520/58241605-bea2fe80-7d66-11e9-937f-ad00ffde4bfe.jpg)

**Comment by keegan-o** [comment]
![image](https://user-images.githubusercontent.com/14983165/82948128-11089380-9f56-11ea-9414-130dc816a6e5.png)
@StanKryvenko Going to need your help with this one, it's not just a animator with positioned bones, it's a series of scripts and I couldn't resolve it with transform positioning in the scene view.

**Comment by keegan-o** [comment]
![image](https://user-images.githubusercontent.com/14983165/82948568-dce1a280-9f56-11ea-8e90-1870d17c2940.png)
Looks pretty messed up in the prefab, not sure where to start with that 😬

**Comment by MazonDel** [comment]
@beastyn Do you have any reproduction steps on this? I've been casting around and doing everything I can think of to break it, but I can't actually get the line to mess up on my end.

**Comment by MazonDel** [comment]
To clarify the problem:
The line visibly "breaks" when the player adjusts their Field Of View (the test-case I've been working with is to set the FOV to 80). This appears to be because the FishingRod is immune to changes in the FOV, while the LineRenderer that connects the Tip of the FishingRod to the Lure is not immune to FOV changes.

Methods Attempted:
 - I have tried to convert the position of the tip into a variety of screen/viewport coordinates and then translate them back into world-space/player-space coordinates, but these consistently returned world-space points for the physical position for where the rod was if it were not FOV immune.
 - I attempted different methods of Raycasting through the camera into the rod, but the results were the same as above.

Notes:
 - It is likely that the required solution will need to take the "Tip" GameObject and split its functionality in two. Currently the same GameObject acts as both the start of the fishing line to the lure AND the end of the fishing line from "Rung2". Despite this dual-use, the Rung2 LineRenderer is unaffected by the FOV-adjusted Tip's position.
 - One possible issue is that the "FPV Renderer" attached to the "Tip" GameObject appears set up differently from the other FishingRod components. It lists "Default-Material (Instance)(FPV) as it's FPV Material, but the other fishing lines list "FishingLine (Instance)(FPV)" as their Material. **Important** I believe that the lack of a shader component of the "curved/standard" for the Tip GameObject is likely the source of the issue.
